Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,27 @@
 import React from 'react';
 import './Header.css';
+
+import HeaderTitle from './HeaderTitle';
+import ImageThumbnail from './ImageThumbnail';
+import HeaderContent from './HeaderContent';
+
+const HeaderContainer = () => {
+    const headerTitleProps = {
+        h1Text: 'Lambda School',
+        spanText: '@LambdaSchool • 28 May'
```

Comment:
Look up the moment() function from moment.js (something you can import into your project) which gives you real time dates on your projects!